pyartcd/pyartcd/pipelines/ocp4.py

```diff
@@ -781,6 +781,14 @@ async def _sync_images(self):
         for record in records:
             if record['has_olm_bundle'] == '1' and record['status'] == '0' and record.get('nvrs', None):
                 operator_nvrs.append(record['nvrs'].split(',')[0])
+        for record in records:
```

Comment:
why a new loop, why not add this logic in the above loop?

Reply:
because we need first to get operator_nvrs list, then remove failed build, we don't know the order so it could be a successful child behind the failed parent so when do operator_nvrs.remove the nvr is not in the list yet

Reply:
Think your comment is about the new loop in line 788. The loop in line 784 seems not needed?

Reply:
updated in https://github.com/openshift-eng/art-tools/pull/814/commits/37f4ee437aab25852342f21f79c978916b784352